semantic-conventions/src/tests/semconv/templating/test_markdown.py

```diff
@@ -133,6 +139,7 @@ def check(
         *,
         expected_name="expected.md",
         extra_yaml_dirs: Sequence[str] = (),
+        extra_yaml_files: Sequence[str] = (),
```

Comment:
I wanted to use a newer `http.yaml` defined in the `metrics_tables` directory. The reason being the `yaml/http.yaml` had older data, and updating it to the newer spec yaml caused other tests to fail. I couldn't have them both parsed in the same test because I would get an error like `Semantic convention 'http' is already defined`. So it wasn't easy to parse the whole `yaml/` directory.

My solution was to be able to specify specific extra yaml files to use for a test, and this solved my problem. 

The root problem is that these test files become out of date quick, we should at some point look at pulling them directly from the spec and so it can be kept up to date more quickly. That way we don't need multiple versions of markdown and yaml laying around, and all the tests run off the same consistent data.

Reply:
> The root problem is that these test files become out of date quick, we should at some point look at pulling them directly from the spec and so it can be kept up to date more quickly.

IMHO, no, this is not what we should do, at least not for these tests. These tests are not designed as integration tests, they are targeted feature tests and it makes no sense to drag in a full spec yaml. These tests should instead use a minimal yaml that just uses group foo with attribute bar instead of actual data. See #63.

Having tests that automatically pull in spec code would *additionally* make sense, but then we don't have e.g. a testMultipleEnum or testRef but instead a testGeneratorIsSpecCompatible (and you'd need to think about how to design the build/change process so that you can also change the output in "incompatible" ways, e.g. temporarily reference a spec PR branch that updates the spec so the semconv table check is in sync again.

----

So consider using something more minimal for this test instead of using a group named "http", maybe use a group "metric_test" or "foo".